src/pages/new/index.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import ThreeParagraphBlock from "../../components/grid-aware/ThreeParagraphBlock";
+import image1 from "../../components/grid-aware/ThreeParagraphBlock/stories/image1.png";
+import image2 from "../../components/grid-aware/ThreeParagraphBlock/stories/image2.png";
+import image3 from "../../components/grid-aware/ThreeParagraphBlock/stories/image3.png";
```

Comment:
I just wanted to explicitly note that I know that I am using the Storybook images here, which is not a good idea, but until we get the final images, I thought it'd be OK to reuse them here.